### PR TITLE
Implement support for calling the mixer's Quota method from mixc.

### PR DESCRIPTION
**Pull request by geeknoid**
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/284)

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -161,7 +161,7 @@ new_git_or_local_repository(
     commit = "acfaa20a074182738494e8fbeb24ab570bda07cb", # Feb 10, 2017 (no releases)
     remote = "https://github.com/istio/api.git",
     # Change this to True to use ../api directory
-    use_local = False,
```
Comment (ZackButcher) [inline WORKSPACE]: We shouldn't include this change.

File: cmd/client/main.go [hunk 2]
```diff
@@ -106,6 +109,9 @@ func withArgs(args []string, errorf errorFn) {
 	rootCmd.PersistentFlags().BoolVarP(&rootArgs.enableTracing, "trace", "", false,
 		"Whether to trace rpc executions")
 
+	rootCmd.PersistentFlags().IntVarP(&rootArgs.repeat, "repeat", "", 1,
```
Comment (ZackButcher) [inline cmd/client/main.go]: Can we add the short-name `r`?

File: cmd/client/quota.go [hunk 3]
```diff
@@ -15,15 +15,98 @@
...
+	}
+
+	var cs *clientState
+	if cs, err = createAPIClient(rootArgs.mixerAddress, rootArgs.enableTracing); err != nil {
+		errorf("Unable to establish connection to %s", rootArgs.mixerAddress)
+		return
+	}
+	defer deleteAPIClient(cs)
+
+	span, ctx := cs.tracer.StartRootSpan(context.Background(), "mixc Check", ext.SpanKindRPCClient)
```
Comment (ZackButcher) [inline cmd/client/quota.go]: Check -> Quota

File: cmd/client/quota.go [hunk 4]
```diff
@@ -15,15 +15,98 @@
...
+			RequestIndex:    int64(i),
+			AttributeUpdate: attrs,
+			Quota:           name,
+			Amount:          amount,
+			DeduplicationId: dedup,
+			BestEffort:      bestEffort,
+		}
+
+		if err = stream.Send(&request); err != nil {
+			errorf("Failed to send Check RPC: %v", err)
```
Comment (ZackButcher) [inline cmd/client/quota.go]: Check -> Quota

File: WORKSPACE (hunk 1 above)
Comment (geeknoid) [inline WORKSPACE]: Yeah, I had to wait for my PR in istio/api to be approved. All done now.

File: cmd/client/main.go (hunk 2 above)
Comment (geeknoid) [inline cmd/client/main.go]: Done.

File: cmd/client/quota.go (hunk 3 above)
Comment (geeknoid) [inline cmd/client/quota.go]: Oops, done.

File: cmd/client/quota.go (hunk 4 above)
Comment (geeknoid) [inline cmd/client/quota.go]: Done.

**Pull request merged**